spec/command/generate_spec.rb

```diff
@@ -17,7 +17,9 @@ def inside_dir(path)
   Dir.chdir original_working_dir
 end
 
-describe Command::Generate do
+describe Command::Generate, :enable_validations, :without_config_file do
```

Comment:
Why do we enable validations?

Reply:
We need to check real case scenario (where validations are enabled).

Otherwise test case for `"when no configuration exist in the project"` context passes even if we don't override `Command::Generate::VALIDATIONS` constant